### PR TITLE
Add: Trading Paints version 2.0.35

### PR DESCRIPTION
**Pull request by Arron17**
- [y] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [y] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [y] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [y] Have you tested your manifest locally with `winget install --manifest <path>`?
- [y] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20325)

### CODE REVIEW

File: manifests/r/RhinodeLLC/TradingPaints/2.0.35/RhinodeLLC.TradingPaints.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: RhinodeLLC.TradingPaints
```
Comment (OfficialEsco) [inline manifests/r/RhinodeLLC/TradingPaints/2.0.35/RhinodeLLC.TradingPaints.installer.yaml:4]:
```suggestion
PackageIdentifier: Rhinode.TradingPaints
```

File: manifests/r/RhinodeLLC/TradingPaints/2.0.35/RhinodeLLC.TradingPaints.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: RhinodeLLC.TradingPaints
```
Comment (OfficialEsco) [inline manifests/r/RhinodeLLC/TradingPaints/2.0.35/RhinodeLLC.TradingPaints.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Rhinode.TradingPaints
```

File: manifests/r/RhinodeLLC/TradingPaints/2.0.35/RhinodeLLC.TradingPaints.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: RhinodeLLC.TradingPaints
```
Comment (OfficialEsco) [inline manifests/r/RhinodeLLC/TradingPaints/2.0.35/RhinodeLLC.TradingPaints.yaml:4]:
```suggestion
PackageIdentifier: Rhinode.TradingPaints
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Could you remove the LLC from the PackageIdentifier? Also remember to rename the Publisher folder from `RhinodeLLC` to `Rhinode`

File: manifests/r/RhinodeLLC/TradingPaints/2.0.35/RhinodeLLC.TradingPaints.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+PackageVersion: 2.0.35
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
```
Comment (OfficialEsco) [inline manifests/r/RhinodeLLC/TradingPaints/2.0.35/RhinodeLLC.TradingPaints.installer.yaml:13]:
```suggestion
  Architecture: x86
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Oh and the application is x86 not x64

**Comment by Arron17** [comment]
Made Suggested changes to change from RhinodeLLC to Rhinode.
Corrected Architecture from x64 to x86
Updated folder name.

**Comment by OfficialEsco** [comment]
Close and reopen the PR (same button), the bot freaked out

**Comment by Arron17** [comment]
Looks like the bot is having issues again, only things in the files that were changed from the previous time when it passed was the name of the publisher folder from "RhinodeLLC" to "Rhinode"
Along with the name updating in all the manifest files.

I've tested the manifest in sandbox and it works perfectly fine.


image

File: manifests/r/Rhinode/TradingPaints/2.0.35/Rhinode.TradingPaints.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+PackageIdentifier: Rhinode.TradingPaints
+PackageVersion: 2.0.35
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```
Comment (OfficialEsco) [inline manifests/r/Rhinode/TradingPaints/2.0.35/Rhinode.TradingPaints.installer.yaml:13]:
```suggestion
  Architecture: x64
```
Lets try this then.. I believe the x86 machine is scuffed cuz its often causing manifests to fail and then not throwing a proper error..

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]
Noted, its not worth messing with x86 Process Programs >.<

**Pull request merged**